solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -4,8 +4,6 @@
 
 public class ServiceFactoryImpl implements ServiceFactory {
 
-
-
     @Override
     public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) throws FxConversionException {
```

Comment:
Line is longer than 120 characters (found 129).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-637656331>*